crypto/engine/eng_linux_getrandom.c

```diff
@@ -0,0 +1,150 @@
+#include <openssl/opensslconf.h>
+
```

Comment:
From here you should probably enclose the below in OPENSSL_SYS_LINUX guards as it is Linux specific.